tests/e2e/test_rosacli_ocm_role.go

```diff
@@ -0,0 +1,187 @@
+package e2e
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-online/ocm-common/pkg/aws/aws_client"
+
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("Edit user role", labels.Feature.UserRole,
```

Comment:
This is more OCM role than user role or ?

Reply:
Will update.